frb_rust/src/handler.rs

```diff
@@ -298,6 +297,21 @@ impl Error {
     }
 }
 
+impl IntoDart for Error {
+    fn into_dart(self) -> allo_isolate::ffi::DartCObject {
+        match self {
+            Error::CustomError(e) => e.box_into_dart(),
+            Error::Panic(panic_err) => match panic_err.downcast_ref::<&'static str>() {
+                Some(s) => s.to_string().into_dart(),
+                None => match panic_err.downcast_ref::<String>() {
+                    Some(s) => s.to_string().into_dart(),
+                    None => "Box<dyn Any>".to_string().into_dart(),
+                },
+            },
+        }
```

Comment:
Seems this code is duplicate with Error.message? Then maybe we can extract a very simple function about this, e.g. `fn error_to_string()`